### PR TITLE
Adding selenium tests

### PR DESCRIPTION
**Pull request by edmoss345**

### CODE REVIEW

**Comment by edmoss345** [comment]
@istride could you review and merge this. Still plenty more to be done on the testing front, but this PR contains some initial tests and examples which should help others if they are coming to write selenium tests. 

Have created a backlog of tests to be added in #1349 , so would like to get this PR merged then next week I will do smaller PRs related to individual groups of tests.

**Review by istride — CHANGES_REQUESTED** [review]
I suggest reducing the number of directories by merging 'questionnaires_common' and 'survey' into one 'questionnaires' directory.

File: home/factories.py [hunk 1]
```diff
@@ -48,7 +49,14 @@ class SVGToPNGMapFactory(DjangoModelFactory):
     class Meta:
         model = SVGToPNGMap
 
+class SurveyFactory(PageFactory):
```
[the remark this answers is not in the document]
Comment (edmoss345) [reply home/factories.py]: Have pointed to the factory in questionnaires app

File: selenium_tests/questionnaires_common/test_questionnaire_inputs.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from selenium_tests.TestBase.base import BaseSeleniumTests
+from selenium.webdriver.support.ui import Select
+from wagtail.core.models import Site
+from questionnaires.models import SurveyFormField
+
+from iogt_users.factories import AdminUserFactory
+from home.factories import ArticleFactory, HomePageFactory, SurveyFactory, SectionFactory
+from wagtail_factories import SiteFactory
+
+class QInputsSeleniumTests(BaseSeleniumTests):
```
[the remark this answers is not in the document]
Comment (edmoss345) [reply selenium_tests/questionnaires_common/test_questionnaire_inputs.py]: have changed

File: selenium_tests/TestBase/base.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import time
+from django.test import LiveServerTestCase
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+
+
+
+class BaseSeleniumTests(LiveServerTestCase):
```
[the remark this answers is not in the document]
Comment (edmoss345) [reply selenium_tests/TestBase/base.py]: have just implemented 'selenium_tests/base.py' for simplicity

File: selenium_tests/example_test/tests.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+from selenium_tests.TestBase.base import BaseSeleniumTests
+
+from wagtail.core.models import Site
+
+from iogt_users.factories import AdminUserFactory
+from home.factories import HomePageFactory
+from wagtail_factories import SiteFactory
+
+class ExampleSeleniumTests(BaseSeleniumTests):
```
[the remark this answers is not in the document]
Comment (edmoss345) [reply selenium_tests/example_test/tests.py]: Have removed and updated readme

File: selenium_tests/homepage/test_login.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        
+    def test_login(self):
+        self.selenium.get('%s%s' % (self.live_server_url, '/accounts/login/'))
+        username_input = self.selenium.find_element_by_name("login")
+        username_input.send_keys(self.user.username)
+        password_input = self.selenium.find_element_by_name("password")
+        password_input.send_keys('test@123')
+        self.selenium.find_element_by_xpath('//button[@type="submit"]').click()
+        body_text = self.selenium.find_element_by_tag_name('body').text
+        assert self.user.username in body_text
```
[the remark this answers is not in the document]
Comment (edmoss345) [reply selenium_tests/homepage/test_login.py]: Have changed, only used this once so far but will bear in mind when writing future tests

**Comment by edmoss345** [comment]
@istride have made the changes as requested, hopefully this is ready to merge. I also took out one of the tests from this PR as it was only partially written so no sense having it in here at this point.

**Review by istride — APPROVED** [review]

**Pull request merged**